wifi-subsys/components/network/src/httpsclient.c

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @brief   Configure a https client to make a request
+ *
+ * @author  RocioRojas <RocioRojas391@gmail.com>
+ */
+
+#include "httpsclient.h"
+#include "esp_event.h"
+#include "esp_log.h"
+#include "esp_netif.h"
+#include "esp_system.h"
+#include "esp_tls.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "nvs_flash.h"
+#include <stdlib.h>
+#include <string.h>
+
+TaskHandle_t http_event_taks;
+
+extern const char
+    server_root_cert_pem_start[] asm("_binary_server_root_cert_pem_start");
+extern const char
+    server_root_cert_pem_end[] asm("_binary_server_root_cert_pem_end");
+
+esp_err_t _http_event_handler(esp_http_client_event_t *evt) {
+    static char *output_buffer;/* Buffer to store response of http
```

Comment:
wrong format for this inline comment
```suggestion
    static char *output_buffer;/* Buffer to store response of http
```